### DISCUSSION TITLE
Handling für gespeicherte Sicherungsdateien *.tar.gz

### DISCUSSION THREAD

**Discussion by Zeppelin-PV** [solectrus/solectrus#3575] [Q&A] [answered]
ich habe es geschafft mit remove in der Kommandozeile nicht nur eine alte Datensicherung zu löschen, sondern gleich die komplette Solectrus Instanz. 
Ist es möglich das Ganze so zu gestalten, dass die Sicherungen über den Raspi Desktop/Filemanager zu löschen sind ?
... muss nun nochmals ran, neu aufsetzen

**Comment by ledermann** [comment, marked as answer]
Ja, mit `rm` muss man aufpassen ;-)

Was die Löschbarkeit angeht: Die [Scripte](https://github.com/solectrus/backup-restore) selbst sind eher nicht die Ursache dafür, dass deine Backups nicht löschbar sind. Das ist vielmehr eine Frage der Nutzerrechte unter Linux.

Die Backups werden im aktuellen Ordner abgelegt. Ein Löschen ist bei Linux grundsätzlich nur für User möglich, die auf den Ordner **Schreibberechtigung** haben. Das hat der von dir verwendete User offenbar nicht.

Um herauszufinden, welcher Berechtigungen auf den aktuellen Ordner gelten:

```bash
ls -ld .
```

Um aktuellen User Schreibberechtigung auf den aktuellen Ordner zu geben, geht das so:

```bash
sudo chmod u+w .
```

Bitte wende den `chmod`-Befehl aber nicht einfach so an. Besser ist, zunächst das dahinter liegende Konzept zu verstehen. Nützlicher Artikel dazu:
https://www.circuitbasics.com/file-permissions-on-the-raspberry-pi/

**Reply by Zeppelin-PV** [reply]
Danke, ich habe nun nochmals neu aufgesetzt. Die Sicherungsdateien zu verwalten, vor allem außerhalb des Raspi zu sichern gelingt mir nun erfolgreich mit dem WindowsTool SCP
Noch eine Frage zur Sicherung: ich habe mitbekommen dass es neue Scripte gibt und habe damit gesichert und auf der zweiten Instanz wieder hergestellt. Die Einstellungen Preise und Einspeißevergütung sowie Texte in dem Bereich werden nicht gesichert ???

**Reply by ledermann** [reply]
Das [gestrige Update der Backup/Restore-Scripte ](https://github.com/orgs/solectrus/discussions/3580) korrigiert genau das: Die Sicherung der PostgreSQL-Datenbank erfolgt nun so, dass ein Wiederherstellen auch gelingt. Preise und Einstellungen werden dann korrekt wiederhergestellt - wenn du hingegen ein Backup wiederherstellst, das **vor** dem Update erstellt wurde, geht das nämlich (stillschweigend) schief. Insofern: Backups wegwerfen, die vor dem gestrigen Update der Scripte erstellt wurden.

Details siehe https://github.com/solectrus/backup-restore/issues/3
